libs/common-system/src/main/java/org/codice/ddf/configuration/PropertyResolver.java

```diff
@@ -13,15 +13,21 @@
  */
 package org.codice.ddf.configuration;
 
+import java.io.FileInputStream;
+import java.io.IOException;
 import java.util.List;
 import java.util.ListIterator;
+import java.util.Properties;
 import org.apache.commons.lang.text.StrSubstitutor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * Class holds a string potentially containing variables of the format ${system.prop} and handles
  * resolving those variables by trying to replace them with system properties
  */
 public class PropertyResolver {
+  private static final Logger LOGGER = LoggerFactory.getLogger(PropertyResolver.class);
```

Comment:
✏️ Nit: Looks like this isn't being used.

Reply:
Opting to keep it in.